### ISSUE TITLE
Focus ring definition

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#1999]
## Description
Define how focus ring is shown in all components

https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html

Some of the componentes currently fail in:

[F78: Failure of Success Criterion 1.4.11, 2.4.7 and 2.4.13 due to styling element outlines and borders in a way that removes or renders non-visible the visual focus indicator](https://www.w3.org/WAI/WCAG22/Techniques/failures/F78)

Vivo study of potential issues:

https://github.com/Telefonica/mistica-design/discussions/2081

We will rely in this technique to avoid SC failure:

https://www.w3.org/WAI/WCAG22/Techniques/general/G165

### Child issues

* #2156 
* #2155 

## Tasks

- [ ] Update specs
- [ ] Android ticket
- [ ] iOS ticket
- [ ] Web ticket

https://github.com/Telefonica/mistica-web/pull/1325

**Comment by aweell** [comment]
### Web


| Component name | Default | Inverse | Alternative | Media |
|--------|--------| ----- | - | - |
| Accordion | ![Image](https://github.com/user-attachments/assets/c539d5d4-a334-4571-b4f5-45d25d11161a) | ![Image](https://github.com/user-attachments/assets/9848d908-55a6-440c-970c-9f22c09f663c) | - | - | 
| Boxed accordion | ![Image](https://github.com/user-attachments/assets/96b9c102-8126-4cdb-95cc-7e7a50f388d8)  | ![Image](https://github.com/user-attachments/assets/c538f085-0559-482b-bb2f-82796824f058) | - | - |
| ButtonPrimary | ![Image](https://github.com/user-attachments/assets/f044204c-8a77-46c5-a9ac-2ab0ff600dcf) | ![Image](https://github.com/user-attachments/assets/1cd26dcd-a6fd-4931-aa39-af67209afce0) | - | - | - |
| Data Card | ![Image](https://github.com/user-attachments/assets/b5a43c71-b838-49ed-bce3-e1b040f5b129) | ![Image](https://github.com/user-attachments/assets/38bf7edb-9a89-45a3-ab88-7c1546491152) | ![Image](https://github.com/user-attachments/assets/54bbce90-c9c0-45d5-a7d0-9d15c72715c0) | 
| Poster Card | ![Image](https://github.com/user-attachments/assets/68f1f743-5cd5-4e5c-9617-91caad6afd44) | ![Image](https://github.com/user-attachments/assets/278cd9b5-c644-4163-b11e-9b3b4d86b84e) | ![Image](https://github.com/user-attachments/assets/631cdba2-b0eb-463c-8116-fb48e84f24d2) |
| Checkbox | ![Image](https://github.com/user-attachments/assets/91305aed-e50f-4100-be08-2bca821363be) | ![Image](https://github.com/user-attachments/assets/1cb031c0-dc98-480e-ace3-ddb61cd12f37) | ![Image](https://github.com/user-attachments/assets/d5042cc2-e7f5-4735-b3aa-c29331b819e2) | ![Image](https://github.com/user-attachments/assets/397e873c-1297-4c09-94f5-ed662a2797e7)
| Chip | ![Image](https://github.com/user-attachments/assets/d4bd21ec-f787-4c53-921c-96527b4d7cbe) | ![Image](https://github.com/user-attachments/assets/3b789821-b33e-4d5c-bea3-a475b9482689) | ![Image](https://github.com/user-attachments/assets/76a161ee-982b-4a66-964e-81ab6ed2438c) | ![Image](https://github.com/user-attachments/assets/b9720821-6283-4095-a425-311a20e0e917)
| Counter | ![Image](https://github.com/user-attachments/assets/0091a7dc-5751-45cb-bbf2-600de6f99a13) ![Image](https://github.com/user-attachments/assets/9c812d3e-7bed-4fe0-91aa-11c50f47053b) | - | - | - |
| Tex field | ![Image](https://github.com/user-attachments/assets/50dbfd6f-66ed-4b32-99bc-3f7893f08be3) | - | - | - |
| Row | ![Image](https://github.com/user-attachments/assets/b0d4ae3c-aa20-4033-b3a5-4bb4957a88fa) | ![Image](https://github.com/user-attachments/assets/98204245-8cb5-4638-a84a-87bd88af9f8a) | ![Image](https://github.com/user-attachments/assets/0b949f0c-dd41-405d-9344-999e9bb7d427) | ![Image](https://github.com/user-attachments/assets/ce0aa718-59b6-4ca7-9789-f37569bfe7cf)
| Boxed row | ![Image](https://github.com/user-attachments/assets/3474c1dc-27e4-477f-9224-93536c01e6a8) | ![Image](https://github.com/user-attachments/assets/efb0ea81-8fec-499e-acd8-f00fff628a4a) | ![Image](https://github.com/user-attachments/assets/988bbc6d-8899-47e9-a706-e277b4ddcb29) | ![Image](https://github.com/user-attachments/assets/cf826e6c-51f8-45e1-9dc5-dd2eac9247c8)
| MenuItem | ![Image](https://github.com/user-attachments/assets/8726fa9c-4e80-4ad2-8a12-264724c1ed8a) | ![Image](https://github.com/user-attachments/assets/bdd46d4f-617d-440d-91c1-047548787506) | ![Image](https://github.com/user-attachments/assets/bf8d2b0a-0f71-4b50-ad9f-5c2f57968687)
| Radio group | ![Image](https://github.com/user-attachments/assets/86977be1-b0de-4e04-a5b8-370f07f204e3) | ![Image](https://github.com/user-attachments/assets/378a6178-d103-4b4b-ad0f-1b7ac5f093f5) | ![Image](https://github.com/user-attachments/assets/91555e44-009e-41cc-adf8-c24b8ec3c5ed) | |
| Tabs | ![Image](https://github.com/user-attachments/assets/37801e11-7f05-4977-9e81-442ab497975c) | ![Image](https://github.com/user-attachments/assets/3e819dcc-3146-4d9c-8b88-157c2ad0d6a2) | ![Image](https://github.com/user-attachments/assets/8e01074f-9995-4957-9e66-7d745d63f06e) | |
| TextLink | ![Image](https://github.com/user-attachments/assets/41c7af1d-c1c5-490d-b96c-a950db354eae) | ![Image](https://github.com/user-attachments/assets/7a2e6e12-a41b-4215-9574-b8201044deea) | ![Image](https://github.com/user-attachments/assets/a962827f-c672-4216-9a79-a81fb5771025) | 





### Other issues

* Slider loses focus ring when moved with arrows
* In safari most of the focus rings in most brand are not visible in inverse (image attached)
* Boxed row don't show focus when has href https://teams.microsoft.com/l/message/19:be8725e15a8c4cc5ae1bf5d67fad4b7f@thread.skype/1740736741144?tenantId=9744600e-3e04-492e-baa1-25ec245c6f10&groupId=bef9be07-03e4-4afb-8a79-f720a63e4c28&parentMessageId=1740736741144&teamName=Novum&channelName=WebCore-public&createdTime=1740736741144 
* Input fields should not hide the border when focused
* [Feedback focus and alert](https://teams.microsoft.com/l/message/19:68854aea147e4787803aff0e380e1208@thread.tacv2/1741878918937?tenantId=9744600e-3e04-492e-baa1-25ec245c6f10&groupId=e265fe99-929f-45d1-8154-699649674a40&parentMessageId=1741878918937&teamName=M%C3%ADstica&channelName=Help&createdTime=1741878918937)

![Image](https://github.com/user-attachments/assets/fa07af5a-1fd1-41ae-b3e0-ae06f1118133)

**Comment by AnaMontes11** [comment]
As an evolution of this issue, we propose implementing **focus ring customisation**, addressing current inconsistencies: the focus ring does not respect the border-radius of the component it is applied to, and its behaviour varies across components.
The goal is to standardise and make the focus ring configurable at the design system level, ensuring it adapts correctly to the shape of each component and behaves consistently.
